cpp/arcticdb/storage/storages.hpp

```diff
@@ -125,30 +124,30 @@ class Storages {
    }
 
   private:
-    VariantStorage& primary() {
-        util::check(!variant_storages_.empty(), "No storages configured");
-        return *variant_storages_[0];
+    Storage& primary() {
+        util::check(!storages_.empty(), "No storages configured");
+        return *storages_[0];
     }
 
-    std::vector<std::unique_ptr<VariantStorage>> variant_storages_;
+    std::vector<std::unique_ptr<Storage>> storages_;
     OpenMode mode_;
 };
 
 inline std::shared_ptr<Storages> create_storages(const LibraryPath& library_path, OpenMode mode, const arcticdb::proto::storage::VariantStorage &storage_config) {
-    using VariantVec = std::vector<std::unique_ptr<VariantStorage>>;
-    VariantVec variants;
-    variants.push_back(create_storage(library_path, mode, storage_config));
+    using StorageVec = std::vector<std::unique_ptr<Storage>>;
```

Comment:
This and the same line in the following function seem pointless, can just use the definition from line 31? (I know you didn't add this, but while we're here we should remove anyway)